Readme.md

```diff
@@ -700,6 +700,85 @@ test
 Environment variables can be configured for options of non-scalar types as well.
 In such case, the values of the environment variable will be split by `Path.PathSeparator` (`;` on Windows, `:` on Linux).
 
+### Suggest mode
+
+Suggest mode provides command-line autocompletion for Powershell and Bash. By default, it is disabled, but it can be enabled as follows:
+
+```csharp
+var app = new CliApplicationBuilder()
+    .AddCommandsFromThisAssembly()
+    .AllowSuggestMode(true) // allow suggest mode
+    .Build();
+```
+
+Once enabled, your shell must be configured to use suggest mode as follows:
+
+1. Add your application to the PATH
```

Comment:
This sounds like a dealbreaker. Can we somehow work around this? As a user, I wouldn't want to add a CLI to PATH just to enable autocompletion...